Containerfile

```diff
@@ -28,18 +23,18 @@ COPY deploy/ deploy/
 COPY hack/ hack/
 COPY internal/ internal/
 
-RUN make build
+RUN GOOS=$TARGETOS GOARCH=$TARGETARCH go build -mod=vendor --ldflags "-s -w" -a -o bin/kubesan cmd/main.go
 
 # CentOS Stream 9 doesn't provide package nbd
 # FROM quay.io/centos/centos:stream9
-FROM quay.io/fedora/fedora:40
+FROM --platform=$TARGETPLATFORM fedora:latest
 
 # util-linux-core, e2fsprogs, and xfsprogs are for Filesystem volume support where
 # blkid(8) and mkfs are required by k8s.io/mount-utils.
-RUN dnf install -qy nbd qemu-img util-linux-core e2fsprogs xfsprogs && dnf clean all
+RUN dnf update -y && dnf install --nodocs --noplugins -qy nbd qemu-img util-linux-core e2fsprogs xfsprogs && dnf clean all
 
```

Comment:
Nice change. I still have a pending patch that will override nbd-client from this line with a version that implements 'nbd-client -i' (aka netlink /sys/block/nbdN/backend support), but that still works with this cleanup in place.